docs/tutorials/tutorial-rollup.md

```diff
@@ -24,18 +24,18 @@ sidebar_label: Aggregate data with rollup
   -->
 
 
-Apache Druid can summarize raw data at ingestion time using a process we refer to as "rollup". Rollup is a first-level aggregation operation over a selected set of columns that reduces the size of stored data.
+Apache Druid&circledR; can summarize raw data at ingestion time using a process known as "rollup". Rollup is a first-level aggregation operation over a selected set of columns that reduces the size of stored data.
 
-This tutorial will demonstrate the effects of rollup on an example dataset.
+This tutorial demonstrates the effects of rollup on an example dataset.
 
-For this tutorial, we'll assume you've already downloaded Druid as described in
-the [single-machine quickstart](index.md) and have it running on your local machine.
+For this tutorial, you should have Druid downloaded as described in
```

Comment:
For consistency, let's introduce a new section called "Prerequisites". See [Query for latest data](https://druid.apache.org/docs/latest/tutorials/tutorial-latest-by#prerequisites) and [Append data](https://druid.apache.org/docs/latest/tutorials/tutorial-append-data#prerequisites) for reference. You can copy and paste the text from the "Prerequisites" section on the "Append data" tutorial. Also, remove the reference to the "Load a file" tutorial since it no longer applies here.

Reply:
[x] "Prerequisites" added
[x] Copy the "Append data" Prerequisites section
[x] remove reference to "Load a file"